Examples/js/earthquakeUSGS.js

```diff
@@ -16,13 +16,8 @@
             alias: "title",
             dataType: tableau.dataTypeEnum.string
         }, {
-            id: "lat",
-            alias: "latitude",
-            dataType: tableau.dataTypeEnum.float
-        }, {
-            id: "lon",
-            alias: "longitude",
-            dataType: tableau.dataTypeEnum.float
+            id: "location",
```

Comment:
Ah yes, good catch, I forgot that was used in the tutorial.  

Thank you! That would be awesome.  Yes, actually if you just clone dev, you can commit directly to it and this PR will be updated automatically.